kafka/pykafka/partition.py

```diff
@@ -36,20 +35,48 @@ def isr(self):
     def topic(self):
         return self._topic
 
-    def fetch_offsets(self, offsets_before, max_offsets=1):
+    def fetch_offset(self, consumer_group):
+        """Use the Offset Commit/Fetch API to get the current offset for this
+            partition for the given consumer group
+
+        :param consumer_group: the name of the consumer group for which to
+            fetch an offset
+        :type consumer_group: str
+        """
+        pass
+
+    def commit_offset(self, consumer_group):
+        """Use the Offset Commit/Fetch API to set the current offset for this
+            partition for the given consumer group
+
+        :param consumer_group: the name of the consumer group for which to
+            fetch an offset
+        :type consumer_group: str
+        """
+        pass
+
+    def fetch_offset_limit(self, offsets_before, max_offsets=1):
```

Comment:
I was going to suggest a better name, but man this API is weird. I have no better suggestions, and this thing is rarely used anyway.